Introduce function checking if reCAPTCHA is enabled. #1197

nosegraze opened the pull request:
#1197

mindctrl commented:
This is because I have recaptcha disabled and the file only loads if it's enabled.

mindctrl commented:
Secondary note, just for consistency I think the function should be named `rcp_is_recaptcha_enabled()`

nosegraze commented:
Updated.

I changed it to always include the `captcha-functions.php` file. I don't think there's any harm in that since we were always checking if it was enabled before running any of the code anyway.
